.github/actions/install_package/action.yaml

```diff
@@ -0,0 +1,34 @@
+name: Install a Python Package
+description: 'Installs a specified Python package from a requirements file, allowing for flexible package management.'
+
+inputs:
+    requirements-path:
+        required: true
+    package:
+        required: true
+
+runs:
+    using: composite
+    steps:
+        -   name: Get project version with yq
+            id: get_python_version
+            uses: mikefarah/yq@v4.46.1
+            with:
+                cmd: yq '.project.requires-python' pyproject.toml
+
+        -   name: Set up Python
```

Comment:
Can you do a check here if python is already installed?

Reply:
Put the installation of python from pyproject.toml into its own action:
https://github.com/SmartMonkey-git/robinson-group-python-template/blob/bebd65e622957bee96e4ca043e0ff5cc1b4468d9/.github/actions/python_from_pyproject/action.yaml

Could you please us it? :)